server/src/main/java/com/cloud/api/ApiResponseHelper.java

```diff
@@ -2515,6 +2515,10 @@ public NetworkResponse createNetworkResponse(ResponseView view, Network network)
                 Domain domain = ApiDBUtils.findDomainById(domainNetworkDetails.first());
                 if (domain != null) {
                     response.setDomainId(domain.getUuid());
+
+                    StringBuilder domainPath = new StringBuilder("ROOT");
+                    (domainPath.append(domain.getPath())).deleteCharAt(domainPath.length() - 1);
+                    response.setDomainPath(domainPath.toString());
```

Comment:
@pierrelefevre 
each domain has a property domainpath. you may use `domain.getPath()` instead


```
(localcloud) 🐱 > list domains filter=path
{
  "count": 2,
  "domain": [
    {
      "path": "ROOT"
    },
    {
      "path": "ROOT/SUBDOMAIN"
    }
  ]
}
```

Reply:
@pierrelefevre 
just double-checked, this PR uses same code in server/src/main/java/com/cloud/api/query/dao/DomainJoinDaoImpl.java
so it is good.

it would be good to extract to a method to avoid duplicated code.